Expensify/App
# Make download links cross-platform

### Fixes
https://github.com/Expensify/ReactNativeChat/issues/287

### Tests
1. Open the app on web, and verify that you see links for Desktop, iOS, and Android in the bottom of the left-hand-nav.
2. Open the app on desktop, and verify that you see links for Web, iOS, and Android in the bottom of the left-hand-nav.
3. Open the app on iOS, and verify that you see only one link in the bottom of the left-hand-nav that says "View on web" (added a couple words to fill space when it's the only link).
